source/data-model.txt

```diff
@@ -0,0 +1,118 @@
+.. _data-model:
+
+===================
+Define a Data Model
+===================
+
+.. contents:: On this page
+   :depth: 1
+
+.. toctree::
+
+   Schemas </schemas>
+   Relationships </schemas/relationships>
+   Values & Secrets </values-and-secrets>
+
+Overview
+--------
+
+Your App's data model is a description of the data that your App uses.
+Your data model helps to ensure that your data is well-formed, contains
+only valid values, and only changes in ways that you expect.
+
+A data model is composed of several components, including:
+
+- A set of declarative schemas that describe your App's data types.
+
+- A set of relationships that define logical connections between your
+  schema.
+
+- A set of static values that you can use to define global constants,
+  environment values, and secret values.
+
+- A set of rule-based query filters that you can use to dynamically
+  limit MongoDB operations to a subset of all data for a given request.
+
+Defining your App's data model is a key part of configuring and using
+Device Sync. You don't need to define a data model to use other App
+Services. However, a data model unlocks many powerful features that can
+save you time and let you focus on building features instead of writing
+boilerplate code.
+
+For example, once you define a data model, you can:
+
+- Automatically generate a GraphQL schema with rich types, queries, and
+  mutations for every document schema. To learn more, see :ref:`GraphQL
+  API <graphql-api>`.
+
+- Automatically generate syncable Realm object models in your native
+  programming language using an App administration tool. To learn more,
+  see :ref:`Generate SDK Object Models <generate-sdk-object-models>`.
+
+Document Schemas
+----------------
+
+A document schema is a JSON object that defines the structure of a
+particular type of data that your App uses. Document schemas control
```

Comment:
This is just a nit, but pedantically, I want something else here because a schema doesn't directly do the controlling/enforcing. It's just a blueprint, which our very complicated rules engine uses to enforce these things. 

```suggestion
particular type of data that your App uses. Document schemas describe
```